types/superagent/superagent-tests.ts

```diff
@@ -219,7 +219,7 @@ request.get("http://example.com/search").retry(2, callback).end(callback);
 })();
 
 // Attaching files
-const blob: Blob = new File([], "thor.png");
+const blob = {} as unknown as Blob;
```

Comment:
```suggestion
declare const blob: Blob;
```
be explicit what it's here (a Blob type object, readonly)